src/Components/CreateImageWizard/steps/Users/component/UserInfo.tsx

```diff
@@ -79,16 +76,19 @@ const UserInfo = () => {
           onChange={(_e, value) => handleNameChange(_e, value)}
           stepValidation={stepValidation}
           fieldName="userName"
+          isEmpty={userName === ''}
```

Comment:
When this gets removed, the input field should still behave the same as the `validated` value is affected by `isPristine`:
```typescript
const [isPristine, setIsPristine] = useState(!value ? true : false);
```

Reply:
I record it with and without empty

https://github.com/user-attachments/assets/d483a8d8-0cf8-4ba3-9cce-d9d3309a5373


https://github.com/user-attachments/assets/a7a69d49-94c2-40a1-ac0d-74b36ae11ffd

Reply:
@regexowl wdyt?

Reply:
Ah, yeah, I see. But that's an error in validation. `useUsersValidation` needs to return an empty error string if `userName` is undefined. So similarly to:
```typescript
      userPassword: !userPassword
        ? ''
        : !isPasswordValid(userPassword)
        ? 'Password must be between 8 and 128 characters, contain at least one uppercase letter, one lowercase letter, and one special character.'
        : '',
```
or the sshKey part. Otherwise you're trying to pass `undefined` to a function that should take only string as an argument.